x-pack/legacy/plugins/apm/public/components/app/Settings/CustomizeUI/CustomLink/index.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiPanel, EuiSpacer, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { isEmpty } from 'lodash';
+import React, { useEffect, useState } from 'react';
+import { CustomLink } from '../../../../../../../../../plugins/apm/server/lib/settings/custom_link/custom_link_types';
+import { useFetcher, FETCH_STATUS } from '../../../../../hooks/useFetcher';
+import { CustomLinkFlyout } from './CustomLinkFlyout';
+import { CustomLinkTable } from './CustomLinkTable';
+import { EmptyPrompt } from './EmptyPrompt';
+import { Title } from './Title';
+import { CreateCustomLinkButton } from './CreateCustomLinkButton';
+
+export const CustomLinkOverview = () => {
+  const [isFlyoutOpen, setIsFlyoutOpen] = useState(false);
+  const [customLinkSelected, setCustomLinkSelected] = useState<
+    CustomLink | undefined
+  >();
+
+  const { data: customLinks, status, refetch } = useFetcher(
+    callApmApi => callApmApi({ pathname: '/api/apm/settings/custom-links' }),
```

Comment:
Not 100% sure but I thought we had a convention of always using underscore in api paths? If that's the case I tthink we should stick to it.

```suggestion
    callApmApi => callApmApi({ pathname: '/api/apm/settings/custom_links' }),
```

Reply:
my bad, don't know why I ended up changing it to underscore.